scripts/linters/pylint_extensions.py

```diff
@@ -818,6 +838,31 @@ def check_newline_above_args(self, node, docstring):
             if line != b'':
                 blank_line_counter = 0
 
+    def check_newline_below_docstring_sections(self, node, docstring):
+        """Checks to ensure that there is a single space below the
+        "Args", "Returns", "Raises", and "Yields" sections in a docstring.
+
+        Args:
+            node: astroid.node.Function. Node for a function or method
+                definition in the AST.
+            docstring: list(str). Function docstring in splitted by newlines.
+        """
+        docstring_sections = {
+            b'Args:': 'single-space-below-args',
```

Comment:
Use `b` before 'single-space-below-args'. Ditto for 3 lines below it.

Reply:
Done!